src/test/java/uk/ac/ebi/eva/runner/JobRestartAsynchronousTest.java

```diff
@@ -76,6 +75,7 @@ public void jobStoppedAndAbandonedCanBeStarted() throws Exception {
         JobExecution jobExecution = launchJob(jobLauncherTestUtils);
         jobOperator.stop(jobExecution.getJobId());
         jobOperator.abandon(jobExecution.getJobId());
+        Thread.sleep(WAIT_FOR_JOB_TO_END);
```

Comment:
Since the thread is  sleeping here and not blocking, I am not sure how **repeatable** these tests are...